### PR TITLE
Added a readme file to try enarx repo

### PR DESCRIPTION
**Pull request by maveraw**
This is an extended version of the video and a tint of perspective about what Enarx does.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# A guide on how to use Try Enarx 🎯
+
+A little amount of efforts are required but we will walk you through all of it. For instance, trying enarx in your own WASM files is a matter of few clicks. Our goal is to understand how it
```
Comment (nickvidal) [inline README.md]: trying Enarx with your own Wasm file...

File: README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# A guide on how to use Try Enarx 🎯
+
+A little amount of efforts are required but we will walk you through all of it. For instance, trying enarx in your own WASM files is a matter of few clicks. Our goal is to understand how it 
+happens step by step by *learning and doing*.
+
+# Few insights on Try Enarx
+* Enarx supports [Webassembly](https://webassembly.org/) and WASM time. It heavily works on it. So try to find out what these terms actually mean to us and why we use it?
```
Comment (nickvidal) [inline README.md]: Enarx supports WebAssembly by using wasmtime as a runtime.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  familiar language. 
+* Choose the language and implements Webassembly to it and then we're good to go( to try enarx with it).
+
+# Steps involed are: (also mentioned in a video)
+
+#### Step 1:  
+Go to [try.enarx.dev](https://try.enarx.dev/)
+
+#### Step 2:  Click on "Start Now" on the Try enarx page and then choose a platform.
+Intel on equinox for window user basically. Other can choose their respective platforms like AMD, Azure etc.
```
Comment (nickvidal) [inline README.md]: You can choose the following platforms: Intel/AMD on Equinix or Azure .

File: README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+#### Step 4: 
+Choose a WASM file or the file you've converted into a WASM file.
+
+#### Step 5:
+ And Deploy🚀
+
+
+## You did it 👏
+
+Tadaa. Now your application is running on a safe ENARX keep. Congrats on your first deploy🔥
```
Comment (nickvidal) [inline README.md]: Enarx Keep

**Review by nickvidal — CHANGES_REQUESTED** [review]

**Comment by maveraw** [comment]
Done @nickvidal

File: README.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+ And Deploy🚀
+
+
+## You did it 👏
+
+Tadaa. Now your application is running on a safe ENARX Keep. Congrats on your first deploy🔥
+Feel free to join us over out [chat](https://chat.enarx.dev/channel/mentorship) and resource to our [Enarx](https://enarx.dev/docs/Start/Introduction) project will help you dive deep into our motive and perspective.
+
+
+
```
Comment (rjzak) [inline README.md:32]: Maybe remove the extra new lines at the end, leaving only one?

File: README.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+#### Step 3:  
+Log in from your Github account.
+
+#### Step 4: 
+Choose a WASM file or the file you've converted into a WASM file.
+
+#### Step 5:
+ And Deploy🚀
+
+
```
Comment (rjzak) [inline README.md:24]: Remove extra line

File: README.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
+# A guide on how to use Try Enarx 🎯
+
+A little amount of efforts are required but we will walk you through all of it. For instance, trying enarx with your own WASM files is a matter of few clicks. Our goal is to understand how it 
+happens step by step by *learning and doing*.
+
+# Few insights on Try Enarx
+* Enarx supports [Webassembly](https://webassembly.org/) by using wasmtime as a runtime. It heavily works on it. So try to find out what these terms actually mean to us and why we use it?
+* Converting your files into WASM file can be easily done with the help of [Webassembly-guide](https://enarx.dev/docs/WebAssembly/Introduction). Also,with the comfort of your 
+  familiar language. 
+* Choose the language and implements Webassembly to it and then we're good to go( to try enarx with it).
```
Comment (rjzak) [inline README.md]: Maybe: `Choose a language which implements Webassembly, then we're good to go.`

File: README.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
+# A guide on how to use Try Enarx 🎯
+
+A little amount of efforts are required but we will walk you through all of it. For instance, trying enarx with your own WASM files is a matter of few clicks. Our goal is to understand how it 
+happens step by step by *learning and doing*.
+
+# Few insights on Try Enarx
+* Enarx supports [Webassembly](https://webassembly.org/) by using wasmtime as a runtime. It heavily works on it. So try to find out what these terms actually mean to us and why we use it?
+* Converting your files into WASM file can be easily done with the help of [Webassembly-guide](https://enarx.dev/docs/WebAssembly/Introduction). Also,with the comfort of your 
+  familiar language.
```
Comment (rjzak) [inline README.md]: Maybe: `language of choice.`

File: README.md (hunk 5 above)
Comment (maveraw) [reply README.md:32]: For sure

**Comment by maveraw** [comment]
@rjzak and @nickvidal Thanks a lot for your time. I will keep these mistakes in mind. Kindly review it once again. Really thankful.

**Comment by rjzak** [comment]
> @rjzak and @nickvidal Thanks a lot for your time. I will keep these mistakes in mind. Kindly review it once again. Really thankful.

I'm not seeing the changes. Maybe squash commits?

**Comment by maveraw** [comment]
image

Aren't u able to see them?

**Comment by rjzak** [comment]
> Aren't u able to see them?

I can see them when I look at the individual commits, but that makes it complicated to understand what the state is of the PR, and generally we require commits to be squashed before being accepted anyway.

**Comment by maveraw** [comment]
Is it all  merged into one now?? @rjzak

**Comment by maveraw** [comment]
Is anyone here ?? @nickvidal @dpal .

File: README.md [hunk 9]
```diff
@@ -0,0 +1,34 @@
+# A guide on how to use Try Enarx 🎯
+
+A little amount of efforts are required but we will walk you through all of it. For instance, trying enarx with your own WASM files is a matter of few clicks. Our goal is to understand how it happens step by step by *learning and doing*.
```
Comment (nickvidal) [inline README.md]: We will walk you through how to try *Try Enarx*. For instance, trying Enarx with your own Wasm file is a matter of few clicks. Our goal is to understand how it happens step by step by *learning and doing*.

File: README.md [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+
+# Few insights on Try Enarx
+* Enarx supports [Webassembly](https://webassembly.org/) by using wasmtime as a runtime. It heavily works on it. So try to find out what these terms actually mean to us and why we use it?
```
Comment (nickvidal) [inline README.md]: * Enarx supports [Webassembly](https://webassembly.org/) by using wasmtime as a runtime. So try to find out what these terms actually mean to us and why we use it?

File: README.md [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+* Converting your files into WASM file can be easily done with the help of [Webassembly-guide](https://enarx.dev/docs/WebAssembly/Introduction). Also,with the language of choice.
```
Comment (nickvidal) [inline README.md]: * Converting your files into a Wasm file can be easily done with the help of the [Webassembly Guide](https://enarx.dev/docs/WebAssembly/Introduction) with the language of your choice.

File: README.md [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+* Choose a language which implements Webassembly, then we're good to go.
+
+# Steps involed are: (also mentioned in a video)
+
+#### Step 1:  
+Go to [try.enarx.dev](https://try.enarx.dev/)
+
+#### Step 2:  
+Click on "Start Now" on the Try enarx page and then choose a platform: Intel/AMD on Equinix or Azure .
```
Comment (nickvidal) [inline README.md]: Click on "Start Now" on the Try enarx page and then choose a platform: Intel/AMD on Equinix or Azure.

File: README.md [hunk 13]
```diff
@@ -0,0 +1,34 @@
...
+
+#### Step 3:  
+Log in from your Github account.
+
+#### Step 4: 
+Choose a WASM file or the file you've converted into a WASM file. And Deploy🚀
```
Comment (nickvidal) [inline README.md]: Choose a Wasm file and Deploy🚀

File: README.md [hunk 14]
```diff
@@ -0,0 +1,34 @@
...
+
+
+## You did it 👏
+
+Congrats on your first deploy🔥. Now your application is running on a safe [ENARX](https://enarx.dev/docs/Start/Introduction) Keep.
```
Comment (nickvidal) [inline README.md]: Congrats on your first deploy🔥. Now your application is running on a safe [Enarx Keep](https://enarx.dev/docs/Start/Introduction).

**Review by nickvidal — CHANGES_REQUESTED** [review]

**Comment by maveraw** [comment]
Hey @nickvidal. Can you have a glance at "Final README"? Is this one correct?

**Pull request closed without merging**